### ISSUE TITLE
Sectionbox cannot calculate the range of sectionbox completely

### ISSUE THREAD

**Issue by AlienwareBoy** [xBimTeam/XbimWebUI#389]
![image](https://user-images.githubusercontent.com/25809878/167976595-5e44ad2d-b328-4e11-b85c-67eb2dbdcf8c.png)

- In viewer, when I put the model into xbim, it can't completely calculate the size of the box model, but only this model can't be displayed. The situation is different in Windows UI and Web UI,WindowUI can be fully displayed, but WebUI cannot calculate the correct boundary value
- In other models, I use the gettargetboundingbox API to get the boundary value of the model, and then use it is successful to use settoplanes API to cut the model
- I tried many models. I feel my equation is correct, but I don't know why only this model appears in the above figure, resulting in the failure of sectioning

**Comment by AlienwareBoy** [comment]
![image](https://user-images.githubusercontent.com/25809878/168005835-3adfb514-c4d4-4081-9a23-1970cf243069.png)

> When I want to cut the front or rear, the view shrinks, and the cutting data shows that it does not form a box. From the view of the viewer, it seems that he did not put the model at the bottom left into the view

**Comment by martin1cerny** [comment]
Does your model contain two or more buildings which are far away from each other?

**Comment by AlienwareBoy** [comment]
> 您的模型是否包含两个或多个彼此远离的建筑物？
Yes, this happens when I load this model that is far away, but when I load one more model on the viewer, the calculation will be correct

**Comment by AlienwareBoy** [comment]
Hello, xbim now supports the product information of the selected model. Can you support the surface information of the selected product? This problem bothers me very much. I want to consult you

**Comment by martin1cerny** [comment]
What kind of surface information would you like to know? The colour? Or the name of the material? Bear in mind that there are often multiple colours and materials per product. Say a door might be made of wood, iron and plastic and its colour might be all brown and white.

**Comment by AlienwareBoy** [comment]
I'm not talking about the material of the product. This information can be obtained through the API. I want to get the geometric information of the product, such as the surface of the product through the API, so that I can associate the surface with some data
















At 2022-05-25 20:11:31, "Martin Černý" ***@***.***> wrote:

What kind of surface information would you like to know? The colour? Or the name of the material? Bear in mind that there are often multiple colours and materials per product. Say a door might be made of wood, iron and plastic and its colour might be all brown and white.

**Comment by martin1cerny** [comment]
This seems to be unrelated to the original topic of this issue. Can you open another one?
